src/client/net/resource.js

```diff
@@ -440,8 +445,17 @@ spf.net.resource.prefetch_ = function(el, type, url, id, group) {
   var isJS = type == spf.net.resource.Type.JS;
   var isCSS = type == spf.net.resource.Type.CSS;
   var doc = el.contentWindow.document;
+  // If an element with a given id already exists, remove it before prefetching
+  // the resource to avoid growing the overall DOM size.  Since `prefetch`
+  // already checks for the element's existence before calling this method,
+  // this is to prevent repeated calls with `opt_force` from always generating
+  // new nodes.
+  var fetchEl = doc.getElementById(id);
```

Comment:
It worked as expected in IE9+, but I found a different issue with IE not re-requesting images that had been loaded before, even if they should be, and I added a work-around.